### DISCUSSION TITLE
Image logo in the left above corner

### DISCUSSION THREAD

**Discussion by hpz24** [letItCurl/minimal_marketing#3] [Ideas]
Hi, 
first of all, your template looks really cool, I love it!

Is there a possibility to use a logo (e.g. png) in the upper left corner?

**Comment by letItCurl** [comment]
Hey! 
Thank for asking ;)

Yes you can!

Just fork the project and work on this line:
https://github.com/letItCurl/minimal_marketing/blob/ad8b62c250a46715d7e6081257b2d2cedf29479a/layouts/partials/navbar.html#L6C9-L6C111

The images are not dynamic in the hugo file because I wanted developer to have the freedom to just code the HTML as they wish :)

**Reply by hpz24** [reply]
Thanks for reply!

I tried with a web-ressource at it works well - but I can´t get it work with a local ressource. 
I checked the documentation and e.g. `src="static/logo.png"` should work - but i get only an 404 error for the file
Is there something different to consider?

**Reply by letItCurl** [reply]
AH yess my friend haha

Look at this: https://github.com/letItCurl/minimal_marketing/blob/ad8b62c250a46715d7e6081257b2d2cedf29479a/layouts/partials/posts.html#L10-L12

When referencing images you need to use the hugo methods ;)

Sorry for the delay, I was distracted with a cool side project :p

What are you building these days?
